Src/FluentAssertions/Equivalency/Execution/CollectionMemberOptionsDecorator.cs

```diff
@@ -32,6 +33,9 @@ public IEnumerable<IMemberMatchingRule> MatchingRules
         get { return inner.MatchingRules.ToArray(); }
     }
 
+    /// <inheritdoc />
+    public IEnumerable<ITypingRule> TypingRules => inner.TypingRules.ToArray();
```

Comment:
🙃 Not sure why we create a copy, but I see we do the same for `MatchingRules` right above, but doesn't for `UserEquivalencySteps` a bit below.